### ISSUE TITLE
date typo in FISH_SAMPLES

### ISSUE THREAD

**Issue by ctsolomon** [MFEh2o/db#82]
In FISH_SAMPLES, for sampleID="WL_WholeShoreline_21070818_1815_AN_MarkRecap.20120228", the year is incorrectly typed as "2107" (instead of 2017) in dateSample and in sampleID as well.

**Comment by kaijagahm** [comment]
I'm expecting this to be caught by #51, but if it's not, I'll come back to it.

**Comment by kaijagahm** [comment]
This is fixed by my updates in #60 .

**Comment by Randinotte** [comment]
@kaijagahm I just ran into this problematic sampleID in FISH_SAMPLES in db version "First Wave". I plan on fixing it in my next update, but it might be important to find out why it didn't get fixed with #60. 

I also found that the metadataID's blgMorphologyDOC.201806_(27 - 30)_ were still in FISH_SAMPLES. Is this related? It could be completely different, but this drag down issue in the metadataID strongly rings a bell and I can't find any documentation on it.

**Comment by kaijagahm** [comment]
You're right that the date issue will be fixed with #60, but I haven't closed out that issue yet--I'm still working on it. So it makes sense that that correction hasn't gone through yet.

I'm less sure about the bluegill dragdown error. I had thought I had fixed that when I added Chelsea's bluegill data to the database, but I'll double check. If you've already fixed those, then fine; otherwise just leave them as is and I'll look into it. It may be that I'm misremembering and this will be corrected in another one of the issues that I haven't finalized yet.
